content/en/synthetics/private_locations/_index.md

```diff
@@ -212,6 +204,10 @@ This command starts a Docker container and makes your private location ready to
     docker-compose -f docker-compose.yml up
     ```
 
+<div class="alert alert-info">You can upload custom root certificates to your private locations to have your API and browser tests perform SSL handshake using your own `.pem` files with the <a href="https://hub.docker.com/r/datadog/synthetics-private-location-worker">Synthetics Private Location Worker</a> version 1.5.3 or later.</div>
```

Comment:
is it expected to have it 2 times?

Reply:
Yes, if you wanted the same message in two tabs. Do you want it to appear in one tab, and if so, which one?